simple-social-icons.php

```diff
@@ -392,26 +406,20 @@ function update( $newinstance, $oldinstance ) {
 		foreach ( $newinstance as $key => $value ) {
 
 			/** Border radius and Icon size must not be empty, must be a digit */
-			if ( ( 'border_radius' == $key || 'size' == $key ) && ( '' == $value || ! ctype_digit( $value ) ) ) {
+			if ( ( 'border_radius' === $key || 'size' === $key ) && ( '' === $value || ! ctype_digit( $value ) ) ) {
 				$newinstance[ $key ] = 0;
 			}
 
-			if ( ( 'border_width' == $key || 'size' == $key ) && ( '' == $value || ! ctype_digit( $value ) ) ) {
+			if ( ( 'border_width' === $key || 'size' === $key ) && ( '' === $value || ! ctype_digit( $value ) ) ) {
 				$newinstance[ $key ] = 0;
-			}
-
-			/** Accept empty colors for permitted keys. */
-			elseif ( in_array( $key, $can_be_transparent, true ) && '' == trim( $value ) ) {
+			} elseif ( in_array( $key, $can_be_transparent, true ) && '' === trim( $value ) ) {
+				// Accept empty colors for permitted keys.
 				$newinstance[ $key ] = '';
-			}
-
-			/** Validate hex code colors */
-			elseif ( strpos( $key, '_color' ) && 0 == preg_match( '/^#(([a-fA-F0-9]{3}$)|([a-fA-F0-9]{6}$))/', $value ) ) {
+			} elseif ( strpos( $key, '_color' ) && 0 === preg_match( '/^#(([a-fA-F0-9]{3}$)|([a-fA-F0-9]{6}$))/', $value ) ) {
+				// Validate hex code colors.
 				$newinstance[ $key ] = $oldinstance[ $key ];
-			}
-
-			/** Sanitize Profile URIs */
-			elseif ( array_key_exists( $key, (array) $this->profiles ) && ! is_email( $value ) && ! 'phone' === $key ) {
+			} elseif ( array_key_exists( $key, (array) $this->profiles ) && ! is_email( $value ) && ! 'phone' === $key ) {
+				// Sanitize Profile URIs.
```

Comment:
`phpcs` wants `elseif` to be right after the closing bracket, not on the next line.

```php
} elseif ( // something here ) {
```